### ISSUE TITLE
Can't reproduce results.

### ISSUE THREAD

**Issue by wl2776** [xiamenwcy/IrisParseNet#17]
Hi!
I cannot reproduce your results.

I cannot run Caffe on my PC, because it is not supported on my system anymore (Ubuntu 22.04 is too new for Caffe).
I've tried both original and your extended-caffe.

Sites www.codewithgpu.com and www.autodl.com are in Chinese language, and I cannot read anything.
Google translator also fails to translate them.

I've tried to convert your model to ONNX format, but several converters have failed, reporting that they don't know parameters, listed in the proto file.

I've managed to port your code to PyTorch.  Model weights have been converted to Torch dictionary using script from https://github.com/vadimkantorov/caffemodel2pytorch
I've checked tensor shapes in debugger, comparing them with `.caffemodel` files loaded in https://netron.app and with the paper.

However, result is wrong even for the image from Codes/Post-processing/data/img. I've got a black image instead of the pupil mask and incorrect iris and iris edge masks.

Here is what I get:
![parse_net_masks](https://github.com/xiamenwcy/IrisParseNet/assets/4774284/fcd9ea75-6e0b-4083-b680-78032a1dfbf9)
Left to right: source image ([001_IP5_IN_R_RI_01_2.JPEG](https://github.com/xiamenwcy/IrisParseNet/blob/master/Codes/Post-processing/data/img/001_IP5_IN_R_RI_01_2.JPEG), converted to BGR and with per-channel mean subtracted), pupil mask, iris mask, iris edge mask.

I suspect that this is due to preprocessing that has been applied to the image.
I've tried to convert the image to BGR format and subtract `(104.00698793, 116.66876762, 122.67891434)`  from blue, red and green channels respectively. 
I've found these values in python scripts, located under `Codes/IrisParseNet/test`
However, the result is still wrong.

What preprocessing do you use?
What data layout does Caffe expect? Is it row-wise or column-wise?

Could you provide your models in ONNX format?

**Comment by wl2776** [comment]
I've found the issue.
When reimplementing layers in PyTorch it is required to create `BatchNorm2D` layers with `track_running_stats=False` parameter. Example: `self.conv1_1_bn = nn.BatchNorm2d(64, momentum=0.95, track_running_stats=False)`
